src/modules/scraper/entry-point/http/controller.ts

```diff
@@ -231,6 +232,21 @@ export class ScraperController {
     }
   }
 
+  @Post("scraper/arb-rebate-reward")
+  @ApiTags("scraper")
+  @Roles(Role.Admin)
+  @UseGuards(JwtAuthGuard, RolesGuard)
+  @ApiBearerAuth()
+  async submitArbRebateRewardJob(@Body() body: OpRebateRewardBody) {
+    const { fromDepositId, toDepositId } = body;
+
+    for (let depositId = fromDepositId; depositId <= toDepositId; depositId++) {
+      this.scraperQueuesService.publishMessage<ArbRebateRewardMessage>(ScraperQueue.ArbRebateReward, {
+        depositPrimaryKey: depositId,
+      });
+    }
+  }
+
```

Comment:
admin endpoint to be able to trigger quickly re-computation of arb rewards, if needed